### PR TITLE
Get Current User for aws and azure

### PR DESCRIPTION
**Pull request by goldblatt**
### Description
Switching up the setup script to remove an azure specific command

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)
Fixes #<issue_number>; Fixes #<issue_number>; Fixes #<issue_number>...

### CODE REVIEW

File: cloud/azure/templates/azure_saml_ses/bin/setup.py [hunk 1]
```diff
@@ -36,6 +36,18 @@ def pre_terraform_setup(self):
...
+    def get_current_user(self):
+        current_user_function = subprocess.run(
+            [
+                "/bin/bash", "-c",
+                f"source cloud/azure/bin/lib.sh && azure::get_current_user_id"
+            ],
+            capture_output=True)
+
+        if current_user_function:
+            current_user = current_user_function.stdout.decode("ascii")
```
Comment (bion) [inline cloud/azure/templates/azure_saml_ses/bin/setup.py:48]: question (blocking): if not should this error out?

**Review by bion — CHANGES_REQUESTED** [review]

**Review by azizova-leyla — APPROVED** [review]

File: cloud/azure/templates/azure_saml_ses/bin/setup.py [hunk 2]
```diff
@@ -36,6 +36,20 @@ def pre_terraform_setup(self):
...
 
+    def get_current_user(self):
+        current_user_function = subprocess.run(
+            [
+                "/bin/bash", "-c",
+                f"source cloud/azure/bin/lib.sh && azure::get_current_user_id"
+            ],
+            capture_output=True)
+        current_user = None
+        if current_user_function:
```
Comment (bion) [inline cloud/azure/templates/azure_saml_ses/bin/setup.py:47]: suggestion (blocking): this `if` will always be true, recommend removing and just checking `current_user` instead

**Review by bion — CHANGES_REQUESTED** [review]

File: cloud/azure/templates/azure_saml_ses/bin/setup.py (hunk 2 above)
Comment (bion) [reply cloud/azure/templates/azure_saml_ses/bin/setup.py:47]: alternatively, could do `if current_user_completed_process.returncode == 0:`

Comment (bion) [reply cloud/azure/templates/azure_saml_ses/bin/setup.py:47]: suggestion (blocking): rename `current_user_function` to `current_user_completed_process`

Comment (goldblatt) [reply cloud/azure/templates/azure_saml_ses/bin/setup.py]: ahh good call!

**Review by bion — APPROVED** [review]

**Pull request merged**